### PR TITLE
add the option to auto-expand output

### PR DESCRIPTION
**Pull request by nightpool**
I want this for my theme :3

I tried a couple of options for passing `config` around, but splitting out an initial call was the best one, since it made managing the defaults much easier

### CODE REVIEW

**Review by AprilSylph — CHANGES_REQUESTED** [review]
breaks current usage of `translate`, should be fixed easily by giving `config` an empty object for a default value too?

**Comment by nightpool** [comment]
breakage is intentional—i think render should be the entrypoint
providing {} as a default config would still be broken
since it doesn't have the default values
i can fix the preview if you want

**Review by AprilSylph — DISMISSED** [review]
looks good! JSDoc for the parameters+returns would be nice but i can do that if you don't feel like it

File: jsondrop.js [hunk 1]
```diff
@@ -6,15 +6,23 @@ const defaultConfig = {
 
 /**
  * Renders a JavaScript object into a pretty HTML JSON-esque representation.
+ * @param {*} value - The value to be rendered. Non-standard objects will only render their object
+ *					  type as a string.
+ * @param {Object} config - The formatter configuration to use. Properties:
```
Comment (AprilSylph) [inline jsondrop.js]: `config` here should be marked as an optional parameter
https://jsdoc.app/tags-param.html#optional-parameters-and-default-values

**Review by AprilSylph — COMMENTED** [review]
JSDoc has syntax for object properties! https://jsdoc.app/tags-param.html#parameters-with-properties

**Review by AprilSylph — APPROVED** [review]
delicious

**Pull request merged**